src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -214,6 +216,7 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
   constructor(
     options: EthereumInternalOptions,
     coinbase: Address,
+    blockTime: BlockTime,
```

Comment:
I prefer to not specify a default in cases such as this, as it encourages more implicit code which is prone to miss-understandings. I also feel that the implementation should be optimised for the "real" use case (while obviously being testable).

I've added a `fromSystemClock()` static constructor. I intentionally didn't just give `getReferenceClockTime` argument a default, because I wanted to make it explicit that it's using the system clock.

I think this gives a nice balance, and also tidies up the real world consumption - where we can just call `fromSystemClock(startTime)` and have the `Date.now` part abstracted away.